axes/middleware.py

```diff
@@ -0,0 +1,30 @@
+from axes.attempts import is_already_locked
+from axes.utils import get_lockout_message
+from django.contrib.auth.backends import ModelBackend
+from django.core.exceptions import PermissionDenied
+
+
+class DjangoAxesAuthBackend(ModelBackend):
```

Comment:
Maybe `AxesModelBackend` would be a neater name?

Reply:
sure! I copy pasted a bunch of this from my subclass implementation and rushed to redact some of the naming and messages